common/tsconfig.json

```diff
@@ -1,9 +1,3 @@
 {
-  "extends": "../tsconfig.base.json",
-  "compilerOptions": {
-    "declaration": true,
-    "declarationDir": "./types",
-    "declarationMap": true,
-    "emitDeclarationOnly": true
-  }
+  "extends": "../tsconfig.base.json"
```

Comment:
In which case, might as well delete all of the generated declaration files, as they will get stale quickly. 
Just delete the `types` folders from host and dashboard.